Formula/p/packetry.rb

```diff
@@ -0,0 +1,54 @@
+class Packetry < Formula
+  desc "Fast, intuitive USB 2.0 protocol analysis application for use with Cynthion"
+  homepage "https://github.com/greatscottgadgets/packetry"
+  url "https://github.com/greatscottgadgets/packetry/archive/refs/tags/v0.1.0.tar.gz"
+  sha256 "8d91ddc17883299f302752b12e11aa539c306304109d733c8863b7bc444c9629"
+  license "BSD-3-Clause"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "gtk4"
+
+  # macOS indirect linkage dependencies
+  on_macos do
+    depends_on "cairo"
+    depends_on "gdk-pixbuf"
+    depends_on "gettext"
+    depends_on "glib"
+    depends_on "graphene"
+    depends_on "harfbuzz"
+    depends_on "pango"
+  end
+
+  # Linux indirect linkage dependencies
+  on_linux do
+    depends_on "cairo"
+    depends_on "gdk-pixbuf"
+    depends_on "glib"
+    depends_on "pango"
+  end
```

Comment:
If they're both macOS and Linux dependencies there is no need to put them in a dedicated block
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => :build
  depends_on "cairo"
  depends_on "gdk-pixbuf"
  depends_on "glib"
  depends_on "gtk4"
  depends_on "pango"

  on_macos do
    depends_on "gettext"
    depends_on "graphene"
    depends_on "harfbuzz"
  end
```